Briancoughlin patch 2 (#256)
Unity-Technologies/com.unity.multiplayer.samples.coop: README.md

| 🛑  IMPORTANT - Early Access 🛑  | 
| -- |

> | Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. 

Reviewer: need closing table
```suggestion
| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package.  |
```

Author: old text returned as per Sams query.

Boss Room is a fully functional co-op multiplayer RPG made with Unity MLAPI. It is built to serve as an educational sample that showcases certain typical gameplay patterns that are frequently featured in similar games.

Our intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.

> __IMPORTANT__:
> - Boss Room supports all major Unity platforms. To use the WebGL platform a custom WebGL transport based on web sockets is needed.

> > - Boss Room is compatible with Unity 2020 LTS onwards.

Reviewer: ```suggestion
> - Boss Room is compatible with Unity 2020 LTS and later.
```

Author: Fix added in edited file

Our intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.

> __IMPORTANT__:

> > - Boss Room supports those platforms supported by MLAPI (Windows and Mac). To use the WebGL platform a custom WebGL transport based on web sockets is needed.

Reviewer 2: Not sure I would mention WebGL, there was zero dev on this

Author: that's the text for MLAPI, was there work on MLAPI on it?

Author: its your readme, you can remove this if you think its not valid

Author: Edited file

> Checkout our Photon-Realtime setup guide, here:
> [Boss Room Photon Setup Guide](Documentation/Photon-Realtime/Readme.md)

> Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs an `9998` external port to be open. 

Reviewer: a `9998` is correct
```suggestion
Alternatively you can use Port Forwarding. The https://portforward.com/ site has guides on how to enable port forwarding on a huge number of routers. Boss Room uses `UDP` and needs a `9998` external port to be open. 
```

Author: we had a lot of thoughts on 'a' or 'an' do you ignore the numbers  or not ....im not sure
